### PR TITLE
PIL - 1233 Assets UI update

### PR DESCRIPTION
**Pull request by vignesha22**
Simulator Screen Shot - iPhone 11 - 2022-02-03 at 05 55 44
Simulator Screen Shot - iPhone 11 - 2022-02-03 at 05 55 54
Simulator Screen Shot - iPhone 11 - 2022-02-03 at 05 56 04
Simulator Screen Shot - iPhone 11 - 2022-02-03 at 05 58 25

### CODE REVIEW

**Comment by IAmKio** [comment]
@paragramm we've recently had trouble with Apple because of NFTs - the NFT graphic above, could you please remove any references to buying, selling, trading and "digital goods" or anything along them lines - we can't use any terms that allude to a user being able to "buy" or make money from NFTs in any way possible or anything that is a "digital asset" or "digital goods". Thanks! cc: @vignesha22

**Comment by vignesha22** [comment]
Simulator Screen Shot - iPhone 11 Pro Max - 2022-02-03 at 18 46 27

Removed NFT banner for empty wallets

File: src/components/lists/ChainListHeader.js [hunk 1]
```diff
@@ -83,16 +80,21 @@ export default ChainListHeader;
...
+  justify-content: center;
 `;
 
-const WalletNotDeployed = styled(Text)`
-  padding-bottom: ${spacing.mediumLarge}px;
-  ${fontStyles.small};
-  color: ${({ theme }) => theme.colors.link};
+const WalletNotDeployed = styled(Icon)`
+  align-items: flex-end;
+  // flex:1;
```
Comment (IAmKio) [inline src/components/lists/ChainListHeader.js:96]: Please remove dead code

File: src/screens/Assets/collectibles/CollectiblesTab.js [hunk 2]
```diff
@@ -50,9 +50,13 @@ import type { Chain } from 'models/Chain';
 // Local
 import { type FlagPerChain, useExpandItemsPerChain } from '../utils';
 import { type CollectibleItem, useCollectibleAssets } from './selectors';
-import { buildCollectibleFromCollectibleItem } from './utils';
+import { buildCollectibleFromCollectibleItem, calculateTotalCollectibleCount } from './utils';
 import CollectibleListItem from './CollectibleListItem';
 
+// assets (Temporarily removed banner for empty wallets until design change)
+// const collectibleBanner = require('assets/images/banners-collectibles.png');
```
Comment (IAmKio) [inline src/screens/Assets/collectibles/CollectiblesTab.js:57]: Same as above, please remove dead code

**Review by IAmKio — CHANGES_REQUESTED** [review]
Minor changes but other than that - LGTM

File: src/screens/Assets/collectibles/CollectiblesTab.js (hunk 2 above)
Comment (vignesha22) [reply src/screens/Assets/collectibles/CollectiblesTab.js:57]: this one is Temporarily removed for NFT banner

**Review by IAmKio — APPROVED** [review]

**Pull request merged**